src/foam/nanos/dig/DIG.js

```diff
@@ -53,6 +53,18 @@ foam.CLASS({
     },
     'cmd',
     'format',
+    {
+      name: 'digColumns',
```

Comment:
I'd rename this to either `columns` or `fields`
I like `fields` better because they're not always rendered as columns, as is the case for a JSON.